intake/catalog/tests/test_local.py

```diff
@@ -362,13 +372,13 @@ def test_flatten_duplicate_error():
 
 
 def test_multi_cat_names():
-    fn = abspath("catalog_union*.yaml")
+    fn = abspath("catalog_union*.yml")
```

Comment:
Was this test wrong in the first place?

Reply:
I believe so unless intake is meant to see .yml as the same as .yaml.

Reply:
They are definitely not meant to be seen as the same, so it's a wonder that this worked before.

Reply:
It didn't fail, just printed an error.